cairo/ethereum/crypto/hash.cairo

```diff
@@ -1,3 +1,20 @@
-from ethereum.base_types import Bytes32
+from ethereum.base_types import Bytes32, Bytes, Uint256
+from src.utils.bytes import bytes_to_bytes8_little_endian
+from starkware.cairo.common.builtin_keccak.keccak import keccak_bigend
+from starkware.cairo.common.cairo_builtins import BitwiseBuiltin, KeccakBuiltin
+from starkware.cairo.common.alloc import alloc
 
 using Hash32 = Bytes32;
+
+func keccak256{range_check_ptr, bitwise_ptr: BitwiseBuiltin*, keccak_ptr: KeccakBuiltin*}(
+    buffer: Bytes
+) -> Hash32 {
+    alloc_locals;
+    let (local dst: felt*) = alloc();
+    bytes_to_bytes8_little_endian(dst, buffer.value.len, buffer.value.data);
```

Comment:
q: in that case no need to return `dst_len` because the value we're passing in other functions is the `buffer.value.len` right?